### PR TITLE
Log duty IDs instead of JSON

### PR DESCRIPTION
**Pull request by moshe-blox**
This PR replaces the huge JSON array in the `got duties` log with a small list of duty IDs.

### CODE REVIEW

File: logging/fields/fields.go [hunk 1]
```diff
@@ -269,3 +269,7 @@ func Errors(val []error) zap.Field {
 func ToBlock(val *big.Int) zap.Field {
 	return zap.Int64(FieldToBlock, val.Int64())
 }
+
+func FormatDutyID(epoch phase0.Epoch, duty *spectypes.Duty) string {
```
Comment (egorka-sereda) [inline logging/fields/fields.go:273]: somewhere on our way we missed these changes https://github.com/bloxapp/ssv/pull/903
I think we need to pull them and them refactor code in current branch regarding top changes we lost.

**Review by egorka-sereda — APPROVED** [review]
After communicating with Mopshe in discord decided to approve changes.
https://discord.com/channels/@me/1061587599694774302/1090644928096391279

File: operator/duties/fetcher.go [hunk 2]
```diff
@@ -106,14 +106,17 @@ func (df *dutyFetcher) updateDutiesFromBeacon(logger *zap.Logger, slot phase0.Sl
 	if len(duties) == 0 {
 		return nil
 	}
-	// print the newly fetched duties
-	var toPrint []serializedDuty
-	for _, d := range duties {
-		toPrint = append(toPrint, toSerialized(d))
+
+	var b strings.Builder
```
Comment (y0sher) [inline operator/duties/fetcher.go:110]: I would note here that if we need to format a duty id list in other places we should move this functionality to `fields.go`. 
@moshe-blox lets merge this!

**Review by y0sher — APPROVED** [review]
go ahead and merge please

File: operator/duties/fetcher.go (hunk 2 above)
Comment (moshe-blox) [reply operator/duties/fetcher.go:110]: agreed, merging..

**Pull request merged**